### ISSUE TITLE
Documentation (The Book): Macros chapter mentions non-existing (?) "lifetime guide"

### ISSUE THREAD

**Issue by RalfJung** [rust-lang/rust#22003]
The chapter on macros <http://doc.rust-lang.org/book/macros.html#hygiene> mentions a "lifetime guide" when using loop labels. As far as I can tell, that guide does not exist anymore (or it has been renamed when becoming a chapter). If that's correct, then loop labels are not mentioned anywhere else in the book, so this subsection is rather hard to follow. If the lifetime guide still exists somewhere, but has been renamed, then the text should be updated (preferably adding a link).

**Comment by steveklabnik** [comment]
It's now the "Ownership Guide"

**Comment by ralfjung-e** [comment]
That guide doesn't seem to explain the loop labels anymore, though.

**Comment by steveklabnik** [comment]
Yes, they're just in the reference now.
